pyteal/ast/__init__.py

```diff
@@ -20,7 +20,7 @@
     Txn,
 )
 from pyteal.ast.gtxn import GtxnExpr, GtxnaExpr, TxnGroup, Gtxn
-from pyteal.ast.block import Block
+from pyteal.ast.block import Block, BlockField
```

Comment:
In general I have concerns about exporting classes that were originally not exported.

What is the justification here?

Reply:
I was unaware of that `BlockField` is not suggested to be exported, but I am not sure how to suppress warning in sphinx on cannot find reference for `BlockField`. Probably need to say something about excluding `BlockField` in searching?

Reply:
I'm a loss for how to solve this without nitpick ignore. If we could somehow disable the nitpick check for our API page, that would be a solution, but I couldn't find support for that type of customization.

If increasing the nitpick ignore list is undesirable, perhaps we should not enable the nitpick option at this time (and defer to a later issue/PR)?

Reply:
same here, I don't see a straightforward solution for suppressing nitpick warning on specific classes, by adding some tag in docstring.

I am somewhat concerned that the nitpick mode is bringing in more complexity than desired correctness, we should probably open a ticket as a reminder about potential space for using nitpick mode in sphinx...

Reply:
after the discussion we decided to close this thread, for we are keeping `__init__.py` unchanged.